OpenOversight/app/static/css/openoversight.css

```diff
@@ -654,3 +654,28 @@ tr:hover .row-actions {
     visibility: hidden;
     width: auto
 }
+
+.red {
+  color: #a00;
+}
+
+.no-box-shadow {
+  box-shadow: none;
+}
+
+.no-border-top {
+  border-top: none;
+}
+
+.no-display {
+  display: none;
+}
+
+.slightly-above {
```

Comment:
This is def kind of an odd name, but CSS IDs/classes always tend to be a bit odd. We're a far way away from having a standard naming convention. These names all work great.